vocabularies/survey-status.ttl

```diff
@@ -0,0 +1,65 @@
+﻿@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix sstatus: <http://linked.data.gov.au/def/survey-status/> .
+
+<http://linked.data.gov.au/def/survey-status> a owl:Ontology .
+
+sstatus:conceptScheme a skos:ConceptScheme;
+  dct:created "2019-04-09T03:31:01.023Z"^^xsd:dateTime;
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  dct:creator "Geological Survey of Queensland" ;
+  skos:definition "The stage in the lifecycle of conducting a survey."@en;
+  skos:prefLabel "Survey Status"@en .
+
+sstatus:intended a skos:Concept;
+  skos:definition "A survey has been proposed, but has not yet commenced."@en;
+  skos:notation "PRO"@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Intended"@en;
+  skos:altLabel "Proposed"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:current-in-progress a skos:Concept;
+  skos:notation "INPRO"@en;
+  skos:definition "A survey is currently in progress."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "In-progress"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:current-on-hold a skos:Concept;
+  skos:notation "ONHOL"@en;
+  skos:definition "A survey is current, but on-hold."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "On-hold"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:interim-report-received a skos:Concept;
+  skos:notation "INRPT"@en;
+  skos:definition "An interim report for the survey has been received."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Interim report received"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:final-report-received a skos:Concept;
```

Comment:
Could this be related to final report and/or data?  For an airborne survey we don't get a final report as minerals doesn't have an activity reporting for final but we do get final data.

Reply:
I've altered the definition to specify that a _report_ may consist of data only in some circumstances.

Reply:
Also, yes, i think open file status should just be handled by the access rights vocab instead of creating composite terminology. Open filing isn't part of the surveying and reporting process, its more an administrative action.

Reply:
Thanks Vance, I agree that this should be controlled by access rights vocab for most objects.  But i was envisaging a status where the input data and output data might be different i.e. gravity data comes in a csv/txt per survey, but is aggregated into a statewide data merge and levelled for publishing.  In this case data may be final received and open access but final product not yet published?

Reply:
Are surveys permanently stopped mid survey for any reason? If so we may need to add a cancelled flag?

Reply:
After discussions with @GSQ-AI suggested of a 'Processed' status where final data has been received in a format (airborne geophysics database 'aseg-gdf') but is further processed to create products (raster grids and images).  Also applies to above gravity example

Reply:
@mark-gordon-dnrme 
Cancelled would be covered by Abandoned. Can add cancelled as altLabel if desired.

@Greenwoodmatthew My initial thought is that the _product_ is something derived from the survey, but is a separate process. 
i.e.
Are products created _after_ the final report/data has been received as aggregations of multiple surveys or value-add products, or
Are there standard workflows for a single survey where it goes 
     intended>in-progress>completed>processed>final